crates/core/tedge_actors/src/actors.rs

```diff
@@ -104,10 +104,10 @@ impl<S: Server> ServerActor<S> {
 /// // Create an actor to handle the requests to a server
 /// let mut calculator_box = SimpleMessageBoxBuilder::new("Calculator - REMOVE ME", 16).build();
 /// let server = Calculator::new(calculator_box);
-/// let actor = ServerActor::new(server, server_box);
+/// let mut actor = ServerActor::new(server, server_box);
 ///
 /// // The actor is then spawn in the background with its message box.
-/// tokio::spawn(actor.run());
+/// tokio::spawn(async move { actor.run().await } );
```

Comment:
I've always wondered what the difference between these 2 variations of spawning an sync function, as I've always used the `async {}` block variant. So, in the previous version where the `actor.run()` was called directly, the `move` was implicit because that function was taking ownership of `self`?